scrapbook-tutorial/src/App.js

```diff
@@ -1,26 +1,110 @@
-import React, { Component } from "react";
+import React, { useState, useEffect, useCallback } from "react";
 import Comment from "./Comment";
 import Form from "./Form";
+import Dialog from "./Dialog";
+import axios from "axios";
+import ApiList from "./ApiList";
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
 
-class App extends Component {
-  state = {
-    characters: [],
+const myDogServerBaseURL = "https://dog.ceo/api";
+
+const App = () => {
+  var previousActiveElement;
+  var dialog;
+  var dialogMask;
+  var dialogWindow;
+
+  var lastFocusableEl;
+  var firstFocusableEl;
+
+  const [comments, setComments] = useState([]);
+  const [items, setItems] = useState([]);
+
+  const loadItems = () => {
+    axios.get(`${myDogServerBaseURL}/breed/hound/list`).then((response) => {
+      console.log("response");
+      console.log(response);
+      setItems(response.data.message);
```

Comment:
Would be good to catch errors here so we can update the ui in case there is an issue.